hack/make-rules/test-e2e-node.sh

```diff
@@ -129,13 +130,26 @@ if [ $remote = true ] ; then
   echo "Ginkgo Flags: $ginkgoflags"
   echo "Instance Metadata: $metadata"
   # Invoke the runner
-  go run test/e2e_node/runner/run_e2e.go  --logtostderr --vmodule=*=4 --ssh-env="gce" \
-    --zone="$zone" --project="$project"  \
-    --hosts="$hosts" --images="$images" --cleanup="$cleanup" \
-    --results-dir="$artifacts" --ginkgo-flags="$ginkgoflags" \
-    --image-project="$image_project" --instance-name-prefix="$instance_prefix" --setup-node="true" \
-    --delete-instances="$delete_instances" --test_args="$test_args" --instance-metadata="$metadata"
-  exit $?
+  if [[ $gubernator = false || $remote = false ]]; then
+    go run test/e2e_node/runner/run_e2e.go  --logtostderr --vmodule=*=2 --ssh-env="gce" \
+      --zone="$zone" --project="$project" --gubernator="$gubernator" \
+      --hosts="$hosts" --images="$images" --cleanup="$cleanup" \
+      --results-dir="$artifacts" --ginkgo-flags="$ginkgoflags" \
+      --image-project="$image_project" --instance-name-prefix="$instance_prefix" --setup-node="true" \
+      --delete-instances="$delete_instances" --test_args="$test_args" --instance-metadata="$metadata"
```

Comment:
Ah, I see the difference is the presence of `| tee build-log.txt`. As long as that goes to the artifacts directory, I think it's safe to always save it.